frontend/app/(profile)/profile/create-project/page.tsx

```
@@ -0,0 +1,284 @@
+              </option>
+            ))}
+          </select>
+          {errors.category_id && (
+            <p className="text-red-500 text-sm mt-1">{errors.category_id}</p>
+          )}
+        </div>
+
+        <div className="mb-4">
+          <label className="block text-gray-700 font-medium mb-2">
+            Required Skills <span className="text-red-500">*</span>
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)